kibana/examples/openshift/values.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+fsGroup: null
+runAsUser: null
```

Comment:
Shouldn't we have the same structure as in `elasticsearch/examples/openshift/values.yaml`?
```
securityContext:
  runAsUser: null

podSecurityContext:
  fsGroup: null
```

Reply:
It would be nice indeed, but trying to keep the charts identical for every change is quite the challenge. For backwards compatibility reasons the Elasticsearch chart does actually still support `fsGroup` directly as per https://github.com/elastic/helm-charts/pull/171. These changes could also be backported to Kibana. Wherever possible I have been trying to keep changes in sync and to follow the same style as other charts but this one slipped in.

Reply:
Isn't this already the case in [kibana/values.yaml#L57-L66](https://github.com/elastic/helm-charts/blob/master/kibana/values.yaml#L57-L66)?

```
podSecurityContext:
  fsGroup: 1000

securityContext:
  capabilities:
    drop:
    - ALL
  # readOnlyRootFilesystem: true
  runAsNonRoot: true
  runAsUser: 1000
```

Reply:
You are 100% right. I could have sworn this was working before my holiday. When I ran `make template` just now it is clearly not doing anything. Just pushed a new commit using the correct values.